lib/state/revisions/reducer.js

```diff
@@ -0,0 +1,16 @@
+import { combineReducers } from 'redux';
+
+const revisionState = ( state = { revision: null, isViewingRevisions: false }, action ) => {
+	if ( 'setRevisionState' !== action.type ) {
+		return state;
+	}
+	console.log( action );
+	return {
+		...state,
+		...action.revisionState,
+	};
+};
+
+export default combineReducers( {
+	revisionState,
+} );
```

Comment:
I've left `combineReducers` pretty much to be ready to eventually split the reducer as you're suggesting.
(Still, would definitely love to have the Redux thing! 😄 )